swagger.json

```diff
@@ -426,6 +426,10 @@
           "type": "string",
           "description": "Round id(s) used in analysis"
         },
+        "applicationCodes": {
```

Comment:
`roundIdList` has to be removed

Reply:
We need roundIdList to fetch courseEmployee Data. Without the roundId we cannot fetch data from UG. UG rest api is not updated yet. I asked Johannes regarding this. He said it will take two months to update UG rest api groups with the application codes. After that we can remove roundIds from here.

Reply:
Done